.travis.yml

```diff
@@ -117,26 +117,26 @@ jobs:
   include:
     # libstdc++
     - { os: "linux", dist: "trusty",  # xenial has libstdc++ from gcc 5.4.0 with newer ABI
-                     env: [ "B2_TOOLSET=gcc-4.8",     "B2_CXXSTD=03,11"    ], addons: *gcc-48    }
+                     env: [ "B2_TOOLSET=gcc-4.8",     "B2_CXXSTD=03,11",    "BOOST_DATE_TIME_TEST_WITHOUT_SELF_CONTAINED_HEADER_TESTS=1" ], addons: *gcc-48    }
     - { os: "linux", dist: "trusty",  # xenial has libstdc++ from gcc 5.4.0 with newer ABI
-                     env: [ "B2_TOOLSET=gcc-4.9",     "B2_CXXSTD=03,11"    ], addons: *gcc-49    }
-    - { os: "linux", env: [ "B2_TOOLSET=gcc-5",       "B2_CXXSTD=03,11"    ], addons: *gcc-5     }
-    - { os: "linux", env: [ "B2_TOOLSET=gcc-6",       "B2_CXXSTD=11,14"    ], addons: *gcc-6     }
-    - { os: "linux", env: [ "B2_TOOLSET=gcc-7",       "B2_CXXSTD=14,17"    ], addons: *gcc-7     }
-    - { os: "linux", env: [ "B2_TOOLSET=gcc-8",       "B2_CXXSTD=17,2a"    ], addons: *gcc-8     }
-    - { os: "linux", env: [ "B2_TOOLSET=gcc-9",       "B2_CXXSTD=17,2a"    ], addons: *gcc-9     }
+                     env: [ "B2_TOOLSET=gcc-4.9",     "B2_CXXSTD=03,11",    "BOOST_DATE_TIME_TEST_WITHOUT_SELF_CONTAINED_HEADER_TESTS=1" ], addons: *gcc-49    }
```

Comment:
so why turn off this test for these?

Reply:
I only left the self-contained header tests enabled for one config on Linux and one on Windows/MSVC to conserve CI time. Note that on Linux, the tests run twice - for C++03 and C++11. Later C++ versions are unlikely to change standard headers as much as between C++03 and C++11.

Reply:
that makes sense

Reply:
Oh, correction - on Linux the tests are enabled on two jobs, not one - on gcc with libstdc++ and clang with libc++.

Reply:
Unless the header tests take too much time, and they shouldn't, it might be worth it to just leave them enabled on all configurations. They catch some obvious mistakes (not just missing includes) without much cost.

Reply:
idk, the CI takes too long already given that typically I've often had to wait hours for results.  Of course most of that is that it doesn't necessarily run right away.  And now we have had one run where a permitted failure wasn't classified right and stop the merge -- and a 'random failure' (no actual unexpected errors in log and passed the last time) stopping the merge...